doc/features/README.md

```diff
@@ -0,0 +1,39 @@
+Classifying tests with feature labels
```

Comment:
this is good but too narrowly focused on the (main) use case. We should focus on the features inspection alone and document the purpose of the  inspection and document the expected schema of the output. Only then we can illustrate how this information can be consumed with the testing use case.